### DISCUSSION TITLE
Exposed port for postgresql

### DISCUSSION THREAD

**Discussion by rquevedo** [Tecnativa/doodba#547] [Q&A]
Hi folks,
I'm trying to back up an instance using external client tools (This is why https://github.com/Tecnativa/doodba/issues/537). I exposed the PostgreSQL port, but i can't make a connection to a partner host with defined port.
The question is, when i expose postgresql port, what would be the connection parameters?

Thanks in advance

**Comment by pedrobaeza** [comment]
How are you exposing such port?

**Reply by rquevedo** [reply]
In the copier answers, this are the relevant lines:

```
postgres_exposed: true
postgres_exposed_port: 5432
```

**Reply by pedrobaeza** [reply]
Correct then. It's supposed to be transparent then if you have a Traefik v2 proxy in your server. That's it, nothing special to put on psql or the tool you use to connect.

**Reply by rquevedo** [reply]
Yes, I suppose that, but this is the problem. These are the domain prod lines in my copier answers:

```
domains_prod:
- hosts:
  - www.partner-host.com
- hosts:
  - partner-host.com
  redirect_to: www.partner-host.com
```

If i do a telnet to www.partner-host.com host, there are no response:

`telnet www.partner-host.com 5432`

Another relevant setting is that instance still use traefik 1.6-alpine. Are those settings (expose port) exclusive to traefik v2?

**Reply by pedrobaeza** [reply]
Yes, as said in my previous answer.

**Reply by rquevedo** [reply]
ok, I understand. Could you please share an inverseproxy.yaml version that's work with traefik v2? I'm still using old one that can be found in doodba scaffolding.
